### PR TITLE
Inspec pivot

### PR DESCRIPTION
**Pull request by dayne**
minitest-handler doesn't appear to be on a pathway to be viable for use in chef13.  They are recommending people move over to using inspec.

This pull request disables the minitest-hanlder in `.kitchen.yml` and turns on some basic inspec smoke tests that just verify packer 1) exists and 2) does kinda run.  Doesn't do the fancy version or sym link verification the minit-tests did.

This works great for binary installs and gives a basic warm fuzzy the packer was installed on the system and runs.

### CODE REVIEW

File: .kitchen.yml [hunk 1]
```diff
@@ -14,16 +17,22 @@ platforms:
 suites:
   - name: binary_install
     run_list:
-      - recipe[minitest-handler]
+#      - recipe[minitest-handler]
```
Comment (andyrepton) [inline .kitchen.yml]: Please remove the line rather than commenting it out

**Comment by andyrepton** [comment]
Thanks for the addition! Could you please remove the minitest handler rather than leaving it in commented out? We have the git history to let us know where it's gone

**Comment by dayne** [comment]
Sure.  I've finished the thought and removed the mini-tests themselves: both from .kitchen.yml and from the tests directory.   

I just noticed I got carried away when adding source_url and issues_url to metadata and bumped the version.  Didn't mean to add that to this pull request so take note.  I think a minor bump reelects the chef13 `node['kernel']['machine`]` change correctly.

**Comment by andyrepton** [comment]
No worries about bumping the version, saves me needing to do it. Thanks!

**Pull request merged**